src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/OptionsTests.cs

```diff
@@ -596,7 +618,7 @@ private static JsonSerializerOptions GetFullyPopulatedOptionsInstance()
 
         private static void VerifyOptionsEqual(JsonSerializerOptions options, JsonSerializerOptions newOptions)
         {
-            foreach (PropertyInfo property in typeof(JsonSerializerOptions).GetProperties())
+            foreach (PropertyInfo property in typeof(JsonSerializerOptions).GetProperties(BindingFlags.Public | BindingFlags.Instance))
```

Comment:
Introduce variable for properties because the line is long?
e.g. `var properties = typeof(JsonSerializerOptions).GetProperties(BindingFlags.Public | BindingFlags.Instance)`